tss-esapi/tests/coverage.sh

```diff
@@ -16,4 +16,4 @@ tpm2_startup -c -T mssim
 # Install and run tarpaulin #
 #############################
 cargo install cargo-tarpaulin
-cargo tarpaulin --features integration-tests --tests --out xml --exclude-files="tests/*,../*" -- --test-threads=1 --nocapture
+cargo tarpaulin --features integration-tests serde --tests --out xml --exclude-files="tests/*,../*" -- --test-threads=1 --nocapture
```

Comment:
```suggestion
cargo tarpaulin --features integration-tests,serde --tests --out xml --exclude-files="tests/*,../*" -- --test-threads=1 --nocapture
```

Either a comma or enclose with quotes, either way, it's better to test this :sweat_smile:

Reply:
fixed